tests/test_parser.py

```diff
@@ -36,14 +41,18 @@ def test_cuts_date(self):
         cuts = Cuts(self.cube).parse('foo:2015-01-04')
         assert cuts[0][2] == date(2015, 1, 4), cuts
 
+    def test_cuts_int(self):
+        cuts = Cuts(self.cube).parse('foo:2015')
+        assert cuts[0][2] == 2015, cuts
+
+    def test_cuts_int_prefixed_string(self):
+        cuts = Cuts(self.cube).parse('foo:2015M01')
+        assert cuts[0][2] == '2015M01', cuts
+
     @raises(QueryException)
     def test_cuts_invalid(self):
         Cuts(self.cube).parse('f oo:2015-01-04')
 
-    def test_cuts_null(self):
```

Comment:
Not sure. I think I removed it in https://github.com/openspending/babbage/pull/14/commits/2e79c1e78fae57889b459942d770af6171ac54bf because it was failing in master.